.gitignore

```
@@ -30,7 +30,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # env files (can opt-in for committing if needed)
-.env*
+.env
```

This will not protect:
```
.env.local
.env.development.local
```

```suggestion
.env*
!.env.example
```

---

updated it.